exampleSite/content/1-main-items.md

```diff
@@ -0,0 +1,42 @@
++++
+fragment = "items"
+#disabled = false
+date = "2017-10-04"
+lastmod = "2017-10-04"
+weight = 100
```

Comment:
As mentioned we can use weight as before and can remove the <weight> from the filename.